core-web/libs/dotcms-scss/jsp/scss/backend/dot-admin/portlets/_view-workflow.scss

```diff
@@ -79,10 +79,6 @@
         padding-left: $spacing-1 * 2;
         margin-bottom: $spacing-1 * 4;
     }
-}
-
-.view-actions__permissions,
-.view-actions__sub-actions {
```

Comment:
Where this goes?

Reply:
The deleted selector is a copy of line 74-75 so, the content of the deleted selector was merge into the other one